src/NuGet.Clients/PackageManagement.UI/UserInterfaceService/INuGetUIContext.cs

```diff
@@ -22,7 +22,7 @@ public interface INuGetUIContext
 
         IOptionsPageActivator OptionsPageActivator { get; }
 
-        IEnumerable<NuGetProject> Projects { get; }
+        IEnumerable<NuGetProject> Projects { get; set; }
```

Comment:
Projets is now read-write since we need to update the list after project add/remove/rename.